libsolidity/codegen/YulUtilFunctions.cpp

```diff
@@ -2333,3 +2333,34 @@ string YulUtilFunctions::extractReturndataFunction()
 	});
 }
 
+string YulUtilFunctions::copyConstructorArgumentsToMemoryFunction(
+	ContractDefinition const& _contract,
+	string const& _creationObjectName
+)
+{
+	string functionName = "copy_constructor_arguments_" + _creationObjectName;
```

Comment:
Can you also add the contract ast id to the function name?

Reply:
I wasn't adding it because it should be a part of creation object name, but will decouple it and do it properly here

Reply:
It's fine that YulUtilFunctions does not know about the association between object names and contracts, but I would feel safer if the constructor ID would still be part of the function name.

Reply:
I added constructor id as well if it is there, and also didn't use _creation object name in naming, instead added name and id separetely...
Now should be much more safe